doc/connectivity/bluetooth/api/mesh.rst

```diff
@@ -5,7 +5,10 @@ Bluetooth Mesh Profile
 
 The Bluetooth mesh profile adds secure wireless multi-hop communication for
 Bluetooth Low Energy. This module implements the
-`Bluetooth Mesh Profile Specification v1.0.1 <https://www.bluetooth.com/specifications/mesh-specifications/>`_.
+`Bluetooth Mesh Profile Specification v1.0.1 <https://www.bluetooth.com/specifications/specs/mesh-profile-1-0-1/>`_.
+
+Implementation of the draft `Bluetooth Mesh Protocol Specification v1.1 <https://www.bluetooth.com/specifications/specs/mesh-protocol-1-1/>`_
```

Comment:
Something with rendering here

Reply:
Sorry, now it looks fine. In one of the previous builds it didn't render the link correctly.